vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -41,6 +42,26 @@ import hudson.AbortException
     'verbose'
 ]
 @Field Set STEP_CONFIG_KEYS = GENERAL_CONFIG_KEYS.plus([
+
+    /**
+     * Additional pod specific configuration. Map with the properties names
+     * as key and the corresponding value as value. The value can also be
+     * a nested structure.
+     * The properties will be added to the pod spec inside node `spec` at the
+     * same level like e.g. `containers`.
+     * This property provides some kind of an expert mode. Any property
+     * which is not handled otherwise by the step can be set. It is not
+     * possible to overwrite e.g. the `containers` property or to
+     * overwrite the `securityContext` property.
+     * Alternate way for providing `additionalPodProperties` is via
+     * `general/jenkinsKubernetes/resources` in the project configuration.
```

Comment:
IMHO this should be `general/jenkinsKubernetes`

Reply:
Adjusted accordingly. It is now `general/jenkinsKubernetes/additionalPodProperties`. Was a cut N paste mistake.

@kaylinche : would you expect `general/jenkinsKubernetes` or is it fine for you to have the long form `general/jenkinsKubernetes/additionalPodProperties`, which is more appropriate IMO?